### PR TITLE
Added `mdsvex` module

### LINKED ISSUE #189: Use `mdsvex`
Editing the HTML in text pages requires to communicate internally with markdown but convert to Svelte markup. To speed up this process we could use [mdsvex](https://mdsvex.com/).

### PR DESCRIPTION
**Pull request by saabi**
Closes #189

### CODE REVIEW

File: ui/rollup.config.js [hunk 1]
```diff
@@ -9,6 +9,8 @@ import resolve from '@rollup/plugin-node-resolve';
 import svelte from 'rollup-plugin-svelte';
 import yaml from '@rollup/plugin-yaml';
```
Comment (mindrones) [inline ui/rollup.config.js]: minor, no extra blank line needed here

File: ui/src/node_modules/app/components/md/a.svelte [hunk 2]
```diff
@@ -0,0 +1,20 @@
+<script>
+    import Link from '@svizzle/ui/src/Link.svelte';
```
Comment (mindrones) [inline ui/src/node_modules/app/components/md/a.svelte]: we need to use tabs everywhere rathenr than spaces

File: ui/src/node_modules/app/components/md/h5.svelte [hunk 3]
```diff
@@ -0,0 +1,8 @@
+<h4 {...$$props}><slot/></h4>
```
Comment (mindrones) [inline ui/src/node_modules/app/components/md/h5.svelte]: minor, I'd have the slot on a new line for readability in these components

File: ui/src/routes/Accessibility.svx [hunk 4]
```diff
@@ -0,0 +1,104 @@
+<script>
+	import CTA from 'app/components/CTA.svelte';
+	import LinkButton from '@svizzle/ui/src/LinkButton.svelte';
+
+	import {
+		contactEmail,
+		toolName
+	} from 'app/config';
```
Comment (mindrones) [inline ui/src/routes/Accessibility.svx]: I think these can go on a single line

File: ui/src/routes/Index.svx [hunk 5]
```diff
@@ -0,0 +1,36 @@
+<script>
+	const repoUrl = 'https://github.com/nestauk/eurito_indicators';
```
Comment (mindrones) [inline ui/src/routes/Index.svx]: I'd use a frontmatter

File: ui/src/routes/Methodology.svx [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+	const DelgadoEtAlUrl = 'https://www.nber.org/papers/w20375.pdf';
+	const eurostatUrl = 'https://ec.europa.eu/eurostat';
+	const HausmanAndHidalgoUrl = 'https://www.pnas.org/content/106/26/10570';
+	const hesaUrl = 'https://www.hesa.ac.uk';
+	const MateosGarciaUrl = 'https://osf.io/preprints/socarxiv/3cu67';
+	const nutsUrl = 'https://ec.europa.eu/eurostat/web/gisco/geodata/reference-data/administrative-units-statistical-units/nuts';
+	const onsUrl = 'https://ons.gov.uk';
+	const patstatUrl = 'https://www.epo.org/searching-for-patents/business/patstat.html';
+	const ukriUrl = 'https://www.ukri.org/';
+</script>
```
Comment (mindrones) [inline ui/src/routes/Methodology.svx]: I'd use a frontmatter

File: ui/src/routes/_layout.svelte [hunk 7]
```diff
@@ -9,7 +9,9 @@
 	import MultiBanner from 'app/components/MultiBanner.svelte';
 	import Nav from 'app/components/Nav.svelte';
 	import NoScript from 'app/components/NoScript.svelte';
-	import Privacy from 'app/components/content/info/Privacy.svelte';
+
```
Comment (mindrones) [inline ui/src/routes/_layout.svelte]: I'd remove this extra blank line

File: ui/src/node_modules/app/components/CTA.svelte [hunk 8]
```diff
@@ -0,0 +1,23 @@
+<script>
+   	import {_screen} from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
```
Comment (mindrones) [inline ui/src/node_modules/app/components/CTA.svelte]: we need to use tabs only

File: ui/src/node_modules/app/components/CTA.svelte [hunk 9]
```diff
@@ -0,0 +1,23 @@
...
+
+<style>
+    .cta {
+		align-content: center;
+		display: flex;
+		flex-direction: column;
+		justify-content: space-around;
+		margin: 2rem 0 3rem 0;
+		row-gap: 1em;
+	}
```
Comment (mindrones) [inline ui/src/node_modules/app/components/CTA.svelte]: this block needs indentation

File: ui/src/node_modules/app/components/CTA.svelte [hunk 10]
```diff
@@ -0,0 +1,23 @@
+<script>
```
Comment (mindrones) [inline ui/src/node_modules/app/components/CTA.svelte]: I'd probably call this component `ResponsiveFlex`, which seems generic enough and could be then ported to `@svizzle/ui`

File: ui/src/routes/guides/indicators.svelte [hunk 11]
```diff
@@ -1,11 +1,13 @@
...
-	import IndicatorsSmall
-		from 'app/components/content/guides/IndicatorsSmall.svelte';
+	import CTA from 'app/components/CTA.svelte';
+	import IndicatorsMedium from './IndicatorsMedium.svx';
+	import IndicatorsSmall from './IndicatorsSmall.svx';
+	import LinkButton from '@svizzle/ui/src/LinkButton.svelte';
 
 	import {toolName} from 'app/config';
+	import theme from 'app/theme';
+
```
Comment (mindrones) [inline ui/src/routes/guides/indicators.svelte]: extra blank line to be removed

File: ui/src/routes/guides/indicators.svelte [hunk 12]
```diff
@@ -1,11 +1,13 @@
 <script>
 	import {_screen} from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
-	import IndicatorsMedium
-		from 'app/components/content/guides/IndicatorsMedium.svelte';
-	import IndicatorsSmall
-		from 'app/components/content/guides/IndicatorsSmall.svelte';
+	import CTA from 'app/components/CTA.svelte';
+	import IndicatorsMedium from './IndicatorsMedium.svx';
+	import IndicatorsSmall from './IndicatorsSmall.svx';
+	import LinkButton from '@svizzle/ui/src/LinkButton.svelte';
```
Comment (mindrones) [inline ui/src/routes/guides/indicators.svelte]: we need to visually separate npm imports from local imports

File: ui/src/routes/index.svelte [hunk 13]
```diff
@@ -1,17 +1,11 @@
 <script>
-	import {_screen} from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
-	import Link from '@svizzle/ui/src/Link.svelte';
+	import CTA from 'app/components/CTA.svelte';
 	import LinkButton from '@svizzle/ui/src/LinkButton.svelte';
```
Comment (mindrones) [inline ui/src/routes/index.svelte]: we need to visually separate npm imports from local imports

File: ui/src/routes/methodology.svelte [hunk 14]
```diff
@@ -1,28 +1,14 @@
 <script>
 	import {_screen} from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
 	import Download from '@svizzle/ui/src/icons/feather/Download.svelte';
-	import Link from '@svizzle/ui/src/Link.svelte';
 	import LinkButton from '@svizzle/ui/src/LinkButton.svelte';
+	import CTA from 'app/components/CTA.svelte';
+
```
Comment (mindrones) [inline ui/src/routes/methodology.svelte]: extra blank line to be removed

**Review by mindrones — CHANGES_REQUESTED** [review]
first pass, mostly just formatting

File: ui/rollup.config.js [hunk 15]
```diff
@@ -8,10 +8,13 @@ import replace from '@rollup/plugin-replace';
 import resolve from '@rollup/plugin-node-resolve';
 import svelte from 'rollup-plugin-svelte';
 import yaml from '@rollup/plugin-yaml';
+import {mdsvex} from 'mdsvex';
 
 import config from 'sapper/config/rollup.js';
 import pkg from './package.json';
 
+import {unescape_code} from './src/node_modules/app/utils/unescape-inlineCode';
```
Comment (mindrones) [inline ui/rollup.config.js]: minor, we need to remove the extra line and sort by filepath

File: ui/src/node_modules/app/components/ResponsiveFlex.svelte [hunk 16]
```diff
@@ -0,0 +1,23 @@
+<script>
+   	import {_screen} from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
```
Comment (mindrones) [inline ui/src/node_modules/app/components/ResponsiveFlex.svelte]: there's a mix of tab and spaces here

File: ui/src/node_modules/app/components/ResponsiveFlex.svelte [hunk 17]
```diff
@@ -0,0 +1,23 @@
...
+</script>
+
+<div class='cta {$_screen?.classes}'>
+	<slot/>
+</div>
+
+<style>
+	.cta {
```
Comment (mindrones) [inline ui/src/node_modules/app/components/ResponsiveFlex.svelte]: I'd rename this class to `ResponsiveFlex` to ease DOM inspection

File: ui/src/node_modules/app/components/md/_layout.svelte [hunk 18]
```diff
@@ -0,0 +1,27 @@
+<script context="module">
+   	import {_screen} from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
```
Comment (mindrones) [inline ui/src/node_modules/app/components/md/_layout.svelte]: there's a mix of tab and spaces here

File: ui/src/node_modules/app/components/md/a.svelte [hunk 19]
```diff
@@ -0,0 +1,20 @@
...
+	import Link from '@svizzle/ui/src/Link.svelte';
+	import appTheme from 'app/theme';
+  
+	const theme = {
+	  color: appTheme.colorLink,
+	  iconStroke: appTheme.colorLink
+	};
+
+	export let href;
+	$: type = href && href.startsWith('http') && 'external' || null;
```
Comment (mindrones) [inline ui/src/node_modules/app/components/md/a.svelte]: I'd add an extra line in between

File: ui/src/node_modules/app/components/md/code.svelte [hunk 20]
```diff
@@ -0,0 +1,7 @@
+<code {...$$props}><slot/></code>
```
Comment (mindrones) [inline ui/src/node_modules/app/components/md/code.svelte]: I'd indent the slot here

File: ui/src/node_modules/app/components/md/h1.svelte [hunk 21]
```diff
@@ -0,0 +1,9 @@
+<h1 {...$$props}>
+	<slot/>
+</h1>
+
+<style>
+ 	h1 {
```
Comment (mindrones) [inline ui/src/node_modules/app/components/md/h1.svelte]: there's a mix of tab and spaces here

File: ui/src/node_modules/app/components/md/h2.svelte [hunk 22]
```diff
@@ -0,0 +1,11 @@
+<h2 {...$$props}>
+	<slot/>
+</h2>
+
+<style>
+ 	h2 {
```
Comment (mindrones) [inline ui/src/node_modules/app/components/md/h2.svelte]: there's a mix of tab and spaces here

File: ui/src/node_modules/app/utils/unescape-inlineCode.js [hunk 23]
```diff
@@ -0,0 +1,18 @@
...
+  return function (tree) {
+    visit(tree, 'inlineCode', unescape);
+
+    function unescape(node) {
+      for (let i = 0; i < entites.length; i += 1) {
+        node.value = node.value.replace(entites[i][0], entites[i][1]);
+      }
+    }
+  };
+}
```
Comment (mindrones) [inline ui/src/node_modules/app/utils/unescape-inlineCode.js]: we need to change spaces for tabs here

File: ui/src/node_modules/app/components/md/_layout.svelte [hunk 24]
```diff
@@ -0,0 +1,27 @@
+<script context="module">
```
Comment (mindrones) [inline ui/src/node_modules/app/components/md/_layout.svelte]: I'd rename this dir to `ui/src/node_modules/app/components/mdsvex`

File: ui/src/routes/Accessibility.svx [hunk 25]
```diff
@@ -0,0 +1,102 @@
+<script>
+	import ResponsiveFlex from 'app/components/ResponsiveFlex.svelte';
+	import LinkButton from '@svizzle/ui/src/LinkButton.svelte';
+
+	import {contactEmail,toolName} from 'app/config';
+	import theme from 'app/theme';
+	import {zipUrl} from 'app/utils/assets';
```
Comment (mindrones) [inline ui/src/routes/Accessibility.svx]: we need to split npm imports from local imports here

File: ui/src/routes/guides/a11ymenu.svelte [hunk 26]
```diff
@@ -1,7 +1,10 @@
 <script>
 	import {_screen}
 		from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
```
Comment (mindrones) [inline ui/src/routes/guides/a11ymenu.svelte]: this seems to be unused

File: ui/src/routes/guides/indicators.svelte [hunk 27]
```diff
@@ -1,11 +1,13 @@
...
-		from 'app/components/content/guides/IndicatorsMedium.svelte';
-	import IndicatorsSmall
-		from 'app/components/content/guides/IndicatorsSmall.svelte';
+	import ResponsiveFlex from 'app/components/ResponsiveFlex.svelte';
+	import IndicatorsMedium from './IndicatorsMedium.svx';
+	import IndicatorsSmall from './IndicatorsSmall.svx';
+	import LinkButton from '@svizzle/ui/src/LinkButton.svelte';
 
 	import {toolName} from 'app/config';
+	import theme from 'app/theme';
```
Comment (mindrones) [inline ui/src/routes/guides/indicators.svelte]: we need to split npm/local imports

File: ui/src/routes/methodology.svelte [hunk 28]
```diff
@@ -1,28 +1,13 @@
...
 	import {_screen} from '@svizzle/ui/src/gauges/screen/ScreenGauge.svelte';
 	import Download from '@svizzle/ui/src/icons/feather/Download.svelte';
-	import Link from '@svizzle/ui/src/Link.svelte';
 	import LinkButton from '@svizzle/ui/src/LinkButton.svelte';
+	import Methodology from './Methodology.svx';
+	import ResponsiveFlex from 'app/components/ResponsiveFlex.svelte';
 
 	import {toolName} from 'app/config';
 	import {zipUrl} from 'app/utils/assets';
 	import theme from 'app/theme';
```
Comment (mindrones) [inline ui/src/routes/methodology.svelte]: split npm/local imports

**Review by mindrones — CHANGES_REQUESTED** [review]
pass 2:
- mostly formatting
- `ui/src/routes/info/disclaimer.svelte` still needs converting
- I'd move svx files in a dir like [1]

[1]
```
routes/
  _content/
    info/*.svx
    guides/*.svx
    *.svx
```

File: ui/src/node_modules/app/components/md/code.svelte (hunk 20 above)
Comment (saabi) [reply ui/src/node_modules/app/components/md/code.svelte]: The issue here it's that a code block will preserve whitespace. We can do the following however:
```
<code {...$$props}><!--
    --><slot/><!--
--></code>
```

**Review by mindrones — APPROVED** [review]
Great job, good to merge! 💯

**Pull request merged**